cpg_workflows/large_cohort/ancestry_pca.py

```diff
@@ -3,17 +3,64 @@
 
 import hail as hl
 import pandas as pd
-from cpg_utils import Path
+from cpg_utils import Path, to_path
 from cpg_utils.config import get_config
 from gnomad.sample_qc.ancestry import run_pca_with_relateds, assign_population_pcs
 
+from cpg_utils.hail_batch import reference_path
 from cpg_workflows.utils import can_reuse
 
 
 MIN_N_PCS = 3  # for one PC1 vs PC2 plot
 MIN_N_SAMPLES = 10
 
 
+def add_background(
+    dense_mt: hl.MatrixTable,
+    sample_qc_ht: hl.Table,
+    pca_background: dict,
+) -> tuple[hl.MatrixTable, hl.Table]:
+    """
+    Add background dataset samples to the dense MT and sample QC HT.
+    """
+    qc_variants_ht = hl.read_table(
+        str(reference_path('gnomad/predetermined_qc_variants'))
+    )
+    dense_mt = (
+        dense_mt.select_cols().select_rows().select_entries('GT', 'GQ', 'DP', 'AD')
+    )
+    for path in pca_background['datasets']:
+        logging.info(f'Adding background dataset {path}')
+        if to_path(path).suffix == '.mt':
+            mt = hl.read_matrix_table(str(path))
```

Comment:
Would we be able to rename this to `background_mt` or something similar so it's clearer

Reply:
agreed, I;ve changed this to `background_mt` (and `background_vds`) now